grid.css

```diff
@@ -0,0 +1,84 @@
+
+.container {
+   margin-right: auto;
+  margin-left: auto;
+ 
+}
+
+.container:after {
+    content: '\a';
```

Comment:
you should use clear: both. It clears all of floating. Also you should use display: block on container:after. Since u don't define any content on after it is important.